### ISSUE TITLE
Update checker is broken in 1.18.1

### ISSUE THREAD

**Issue by ModernGamingWorld** [Qelifern/IronFurnaces#88]
**Describe the bug**
I have Update Checker settings Disabled in my custom pack, but it keeps telling me in Game that the Update check for this Mod failed, anyway to prevent this

#Update Checker Settings
[updates]

	[updates.check_updates]
		# true = check for updates, false = don't check for updates.
		# Default: true.
		updates = false

**To Reproduce**
1.
2.
3.

**Expected behavior**

**Versions**
- **Iron Furnaces**: 1.18.1-3.1.4
- **Forge**: 39.0.88
- **Fabric**: x.x.x

<details>
  <summary>Crashlog</summary>
  
  ```
  Log goes here

  ```
![2022-02-25_16 42 13](https://user-images.githubusercontent.com/75923060/155812980-2e996889-78bb-4a7d-9d1f-6680a08b8de5.png)

**Comment by faeandsugar** [comment]
Can confirm update checker is broken when set to `false`; still attempts to check and kicks back that there was an error.

**Comment by PernDragon** [comment]
Confirmed same here.

**Comment by pavog** [comment]
Stack trace is:
```
[ Iron Furnaces Update Checker/ERROR] [ironfurnaces.IronFurnaces/]: Update Check failed!
java.io.FileNotFoundException: https://raw.githubusercontent.com/Qelifern/118IronFurnaces/main/update/updateVersions.properties
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1993) ~[?:?]
	at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1589) ~[?:?]
	at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:224) ~[?:?]
	at java.net.URL.openStream(URL.java:1161) ~[?:?]
	at ironfurnaces.update.ThreadUpdateChecker.run(ThreadUpdateChecker.java:28) [ironfurnaces-1.18.1-3.1.4.jar%23158!/:3.1.4]
```

**Comment by Qelifern** [comment]
fixed in 3.2.0
